cfgov/unprocessed/apps/research-reports/css/main.less

```diff
@@ -10,12 +17,12 @@
 .o-report-sidenav {
   &.sticky {
     position: fixed;
+    width: 100%;
     top: 0;
-    width: 240px;
+    .respond-to-min( @bp-lg-min, {
+      max-width: 240px;
+    } );
   }
-  display: block;
-  overflow-y: scroll;
-  height: 100%;
```

Comment:
This code is needed so that on very long ToCs on desktop view, the ToC is scrollable